scripts/test

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n ${RASTER_VISION_DEBUG} ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Run linters and tests.
+"
+}
+
+if [[ ${BASH_SOURCE[0]} == "${0}" ]]; then
+    if [[ ${1:-} == "--help" ]]; then
+        usage
+    else
+        if [[ -z ${CI} ]]; then
+            # Local test suite runs against pytorch image by default
+            IMAGE_TYPE=${IMAGE_TYPE:-pytorch}
+        fi
```

Comment:
This script is run in CI to run the full test suite. It can also be used locally for sake of convenience, and will default to using the pytorch variant.